aptos-move/framework/cached-packages/src/aptos_framework_sdk_builder.rs

```diff
@@ -3677,6 +3684,23 @@ pub fn staking_proxy_set_voter(
     ))
 }
 
+/// Updates the major version to a larger version.
```

Comment:
What are these changes?

Reply:
since existing `version::set_version` is an entrypoint and is referenced in this sdk builder, so should be its async version?